modules/dop/services/workbench/workbench.go

```diff
@@ -95,7 +88,7 @@ func (w *Workbench) GetUndoneProjectItem(userID string, issueSize int, pro apist
 		PageSize: uint64(issueSize),
 		IssueListRequest: apistructs.IssueListRequest{
 			ProjectID:    uint64(pro.ID),
-			StateBelongs: StateBelongs,
+			StateBelongs: apistructs.StateBelongs,
```

Comment:
func `GetUndoneProjectItem` should be deleted?

Reply:
done